### ISSUE TITLE
ImportError: No module named cv2

### ISSUE THREAD

**Issue by YukihimeX** [leblancfg/autocrop#81]
No matter which version of opencv I installed into my canda VirtualEnv (python 3.6 and 3.7) I cannot (for the love of god) make opencv to work.
`import cv2` always give me the "No module name cv2" error.
It's like the damn thing is blind as fck and can't seem to find that opencv is already installed into that Virtual Env. I installed both requirements.txt and requirements-test.txt as well, just to be sure, but still same result. 
Does anyone have any idea how to go about solving this issue?
Which OpenCV version did you guys have that work properly?

**Comment by leblancfg** [comment]
Hi @YukihimeX! Well that sucks...

My first guess would be to assume that you're running autocrop from your virtual environment, but autocrop is using a different `$PYTHONPATH`? Did you activate your environment, and double-check you installed autocrop in your env? (I'm assuming yes, but worth double-checking)

I've done the following steps and cannot duplicate your issue:

1. `conda create -n testenv`
2. `conda activate testenv`
3. `pip install autocrop`
4. `cd folder_with_images`
5. `autocrop`

Works as intended, at least on my Windows box.

Suggested next step:
1. Confirm autocrop was installed to env and that it is activated
2. Try using `conda run autocrop` and post results.

Hopefully that will get us somewhere :)

Note to self: tangentially related to #7.

**Comment by YukihimeX** [comment]
I had some luck since the last time I posted the issue here and somehow manage to get OpenCV to work by installing other users packages that is below 4.0 instead of the conda forge or conda...
But now I get this `ImportError: attempted relative import with no known parent package` error when I run the code. Maybe I've go about it the wrong way? If so please let me know.
Here's my step...
1. Activate Conda Virtual Env
2. cd /d autocrop directory
3. run this code `autocrop.py autocrop -i pics -o crop -r nofaces -w 512 -H 512`

Please take a look at the screenshot and let me know if I did anything wrong.
I created the 3 folders (pics, crop, nofaces) into autocrop/autocrop directory and put a few pictures into the pics folder as a test.
![Autocrop](https://user-images.githubusercontent.com/50353278/71832539-c2269400-3070-11ea-8d77-666513fef178.JPG)

**Comment by leblancfg** [comment]
OK so, two things here. Running the `autocrop.py` file directly is not the same as calling `autocrop`. I think that's why you're getting those weird import errors. When you install autocrop, you're not just getting a file you can then `python autocrop.py` with.

Typically, CLI tools such as `autocrop`, or `awscli`, `cookiecutter`, `pgsql`, etc. are meant to be used in any folder on your machine, and need to be found in your `PATH`. You call then by their name directly in the terminal, e.g. `pgsql -U postgres -d northwind` will start up that utility, no matter where you are in the filesystem.

Taking a step back, you can go back to that folder and try running 

    $ autocrop -i pics -o crop -r nofaces -w 512 -h 512


... or am I reading this all wrong, and that's what you trying previously? 😨

**Comment by YukihimeX** [comment]
Ah I see what you mean... 
Still, when I run `conda run autocrop -i pics -o crop -r nofaces -w 512 -h 512
`
I get `autocrop: error: argument -i/--input: Input folder does not contain any image files` error.

Even though I created those 3 folders into the images directory.
![Autocrop2](https://user-images.githubusercontent.com/50353278/71856317-2b29fe00-30a9-11ea-8ba7-81d65a28cc59.JPG)

Maybe I messed up somewhere else.
